cmd/clusterawsadm/cmd/alpha/migrate/migrate.go

```diff
@@ -94,7 +95,7 @@ Supported versions: %v`, supportedVersions),
 
 	newCmd.Flags().StringVarP(&clusterName, "clusterName", "n", "", "name of existing Cluster object")
 	if err := newCmd.MarkFlagRequired("clusterName"); err != nil {
-		_ = fmt.Errorf("error making required flag")
+		panic(errors.Wrap(err, "error making required flag"))
```

Comment:
This is worth noting, we were swallowing this error before. It should never happen though and I don't think there's a way to surface it without some larger changes.